spec/views/casa_orgs/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe "casa_org/edit", type: :view do
+  before do
+    assign(:contact_type_groups, [])
+    assign(:contact_types, [])
+    assign(:hearing_types, [])
+    assign(:judges, [])
+    assign(:sent_emails, [])
+
+    sign_in build_stubbed(:all_casa_admin)
```

Comment:
The user who can view this is a `casa_admin` (the `all_casa_admin` is the super user that RfG uses to see all the organizations)